app/models/org.rb

```diff
@@ -0,0 +1,20 @@
+class Org < ApplicationRecord
+  validates :handle, presence: true,
+    uniqueness: { case_sensitive: false },
+    length: { within: 2..40 },
+    format: { with: /\A[A-Za-z][A-Za-z_\-0-9]*\z/ }
+  validates :name, presence: true, length: { within: 2..255 }
+  validate :unique_with_user_handle
+
+  def unique_with_user_handle
+    errors.add(:handle, "has already been taken") if handle && User.where("handle = lower(?)", handle.downcase).any?
+  end
+
+  has_many :rubygems, through: :ownerships
```

Comment:
missing `ownerships` association?

Reply:
Whoops, meant to remove this for now. Making `ownerships` polymorphic and adding them to orgs is a future PR.